shared/actions/signup.js

```diff
@@ -84,11 +84,12 @@ function checkInviteCode(
     })
 }
 
-function requestAutoInvite(): TypedAsyncAction<
-  CheckInviteCode | NavigateAppend | NavigateTo | SignupWaiting
```

Comment:
i think its maybe not that useful to type the types of actions our actions will possibly dispatch. dunno how much we actually get by these guarantees.